Rakefile

```diff
@@ -92,7 +93,9 @@ namespace :test do
 end
 
 desc 'Lint the code through pylint'
-task 'lint' => 'ci:default:lint'
+task 'lint' => ['ci:default:lint'] do
+  sh 'rubocop --fail-level W'
```

Comment:
Nice for the rake task! Will fix.
Forgot to stage the rubocop yaml file :fearful: